cves/kernel/CVE-2012-6545.yml

```diff
@@ -55,7 +55,8 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Some memory was not being initialized correctly, which meant the memory was being leaked. A well thought out program could use this to read sensitive information.
```

Comment:
Please add a line break at the 80th character.  The description could use more context.

Reply:
I'll try to give more info.